src/ctim/domain/disposition.cljc

```diff
@@ -0,0 +1,18 @@
+(ns ctim.domain.disposition)
+
+(def ^:private weights
+  {"Clean"      1
+   "Malicious"  2
+   "Suspicious" 3
+   "Unknown"    4})
```

Comment:
This is only ranking the importance, so I guess the assumption is that common is the same as default, which is 99.  If that is wrong, then I should change it.